pkg/services/alerting/store_test.go

```diff
@@ -49,12 +50,12 @@ func TestIntegrationAlertingDataAccess(t *testing.T) {
 		ss := db.InitTestDB(t)
 		tagService := tagimpl.ProvideService(ss, ss.Cfg)
 		cfg := setting.NewCfg()
-		cfg.RBACEnabled = false
 		store = &sqlStore{
 			db:         ss,
 			log:        log.New(),
 			cfg:        cfg,
 			tagService: tagService,
+			features:   featuremgmt.WithFeatures(),
```

Comment:
needed for not getting a nil pointer exception in the Permissions filtering, which checks the NestedFolder feature on line: https://github.com/grafana/grafana/blob/main/pkg/services/sqlstore/permissions/dashboard.go#L208